UMLEditor/UMLEditor/Classes/Diagram.cs

```diff
@@ -146,6 +146,37 @@ public void AddRelationship(string sourceClassName, string destClassName, string
         
     }
 
+    /// <summary>
+    /// Adds a method to a class
+    /// </summary>
+    /// <param name="toClass">Class to add method to</param>
+    /// <param name="returnType">Return type of method</param>
+    /// <param name="methodName">Name of method</param>
+    public void AddMethod(string toClass, string returnType, string methodName)
+    {
+        if (!ClassExists(toClass))
+        {
+            throw new ClassNonexistentException($"Class {toClass} does not exist");
+        }
+        GetClassByName(toClass).AddMethod(returnType, methodName);
+    }
+    
+    /// <summary>
+    /// Adds a method to a class
```

Comment:
We might want to add a mention here to what kind of Add method this is, for example "add method to a class given a list of params" or something.